File: app/src/main/java/com/lifedawn/capstoneapp/common/util/AttendeeUtil.java
package com.lifedawn.capstoneapp.common.util;

import android.content.ContentValues;
import android.content.Context;
import android.provider.CalendarContract;

import com.lifedawn.capstoneapp.common.viewmodel.FriendViewModel;

import java.util.List;

public class AttendeeUtil {
	private AttendeeUtil() {
	}

	public static String toListString(List<String> attendeeList) {
		StringBuilder stringBuilder = new StringBuilder();
		final String divider = ", ";

		final int totalCount = attendeeList.size();
		int count = 1;

		for (String attendee : attendeeList) {
			stringBuilder.append(attendee);
			if (count < totalCount) {
				stringBuilder.append(divider);
			}
			count++;
		}
		return stringBuilder.toString();
	}
}

File: app/src/main/kotlin/com/lifedawn/capstoneapp/util/AttendeeUtil.kt
package com.lifedawn.capstoneapp.util

class AttendeeUtil {
    companion object {
        fun toListString(attendeeList: ArrayList<String>): String {
            val stringBuilder = StringBuilder()
            val divider = ", "
            val totalCount = attendeeList.size
            var count = 1

            for (attendee in attendeeList) {
                stringBuilder.append(attendee)
                if (count < totalCount) {
                    stringBuilder.append(divider)
                }
                count++
            }
            return stringBuilder.toString()
        }
    }

}